.github/workflows/create_hotfix_branch.yml

```diff
@@ -0,0 +1,26 @@
+name: Create new hotfix branch
+
+on:
+  workflow_dispatch:
+    inputs:
+      hotfix_name:
+        description: Hotfix branch name
+        required: true
+
+jobs:
+  create-branch:
+    name: Create new branch
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/master-v2'
+    steps:
+      - name: Set env BRANCH_NAME
+        run: |
+              BRANCH="${{ github.event.pull_request.head.ref }}"
+              echo "BRANCH_NAME=hotfix" >> "$GITHUB_ENV"
```

Comment:
how do you distinquish between v1 nad v1 hotfix branches in order to trigger follow up actions?

Reply:
These actions will be pushed to develop-v2, and develop-v1 will have the current actions.